contracts/pools/tbtc2/DepositTBTC2.vy

```diff
@@ -0,0 +1,366 @@
+# @version 0.2.7
+"""
+@title "Zap" Depositer for Curve tBTCv2 pool
+@author Curve.Fi
+@license Copyright (c) Curve.Fi, 2020 - all rights reserved
```

Comment:
Not an expert on licensing, but shouldn't we update the year to `2021`?

Reply:
Probably. Will take this into account while opening the PR in Curve repo.

Reply:
https://github.com/keep-network/curve-contract/pull/1/commits/7365c9fa5f28bccd24042e385bb34dc973971a54